frontend/app/src/components/import/GroupedImport.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <card>
+    <div class="pa-2">
+      <v-select
+        v-model="selectedSource"
+        :label="$t('import_data.select_source.title')"
+        outlined
+        :items="sources"
+        item-value="identifier"
+        item-text="name"
+        :hide-details="true"
+      >
+        <template #item="{ item }">
+          <v-row v-if="item">
+            <v-col class="d-flex align-center justify-space-between col-12">
+              <div
+                :class="`${$style.image} ${
+                  dark ? $style['image--inverted'] : ''
+                }`"
+              >
+                <v-img
+                  :src="item.logo"
+                  max-height="100%"
+                  max-width="100%"
+                  position="center left"
+                  :contain="true"
+                />
+              </div>
+              <div>{{ item.name }}</div>
+            </v-col>
+          </v-row>
+        </template>
+        <template #selection="{ item }">
+          <div
+            :class="`${$style.image} ${dark ? $style['image--inverted'] : ''}`"
```

Comment:
The same as above here?

Reply:
Hmm actually yes, is there any way to fill same content to those two template?

Reply:
I guess I have to use v-for for this?

Reply:
updated @kelsos